server/routes/auth.js

```diff
@@ -0,0 +1,16 @@
+const passport = require('../auth');
+const router = require('express').Router();
+
+router.post('/auth/login', passport.authenticate('local'),
+  (req, res) => {
+    return res.status(200).json({
```

Comment:
`res.json()` automatically sets HTTP code to 200 so `.status(200)` is redundant.